Sources/ProjectSpec/Scheme.swift

```diff
@@ -141,6 +141,7 @@ extension Scheme.Test: JSONObjectConvertible {
 
     public init(jsonDictionary: JSONDictionary) throws {
         config = try jsonDictionary.json(atKeyPath: "config")
+        gatherCoverageData = try jsonDictionary.json(atKeyPath: "gatherCoverageData") ?? false
```

Comment:
Sorry just saw this. Is there a warning that the try isn’t needed? Optional json(atKeyPath:) properties are non throwing. I just want to make sure this doesn’t fail if the property isn’t there

Reply:
Good point, try isn't necessary, removed. 👍